docs/projects/tad/reporting-standard/latest.md

```diff
@@ -113,10 +113,23 @@ legal base the following fields are present.
     2. `link` (OPTIONAL, string). URI pointing towards the contents of the law.
 17. `used_data` (OPTIONAL, string). An overview of the data that is used in the system.
 18. `technical_design` (OPTIONAL, string). Description on how the system works.
-19. `external_providers` (OPTIONAL, list[string]). Name of an external provider, if relevant. There can
-be multiple external providers.
+19. `external_providers` (OPTIONAL, list). If relevant, these fields allow to store information on external providers.
+ There can be multiple external providers.
+    1. `name` (OPTIONAL, string). Name of the external provider.
+    2. `version` (OPTIONAL, string). Version of the external provider reflecting its relation to previous versions.
```

Comment:
I am not sure if a version makes a lot of sense here. This field was included because of the algoritmeregister, where a field "external provider" is present. Usually this is filled with the name of a company, not a specific product. On the other hand, since all these fields are optional, it's maybe also ok to provide a version field. I am not sure though.

Reply:
I understand your consideration. The reason this is proposed to be included is that we than give space to document all information of the general description of the EU AI Act in one go.